Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -145,11 +236,17 @@ void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
       scatteringInRegion2 *=
           m_config.sigmaScattering * m_config.sigmaScattering;
 
+      /// this is an approximate worst case multiple scattering term assuming
+      /// the lowest  pt we allow and the estimated theta angle
+      float sigmaSquaredScatteringMinPt =
+          iSinTheta2 * m_config.kConstant /
+          (std::abs(m_config.minPt) * std::abs(m_config.minPt));
```

Comment:
is there a reason you didn't use std::pow(m_config.minPt,2) like you did further below? note that the std::abs is not necessary here.

Reply:
Isn't `x*x` usually preferable over `pow(x, 2)`?